docs/source/geometry.conversions.rst

```diff
@@ -36,7 +36,7 @@ Homography
 Quaternion
 ----------
 
-.. autofunction:: quaternion_to_angle_axis
+.. autofunction:: quaternion_to_axis_angle
```

Comment:
Can you add an alias for the old function with a deprecation warning?

example https://github.com/kornia/kornia/blob/83d78273a78ea995df92b6fb53bfcfad48e75882/kornia/filters/kernels.py#L954

Reply:
+1 And add a not that after 0.7 we deprecate